db/seeds/seedData/mentors.js

```diff
@@ -12,7 +12,8 @@ const mentors = [
     position: 'senior dev',
     dev_type: 'backend',
     stack: 'Ruby/Rails, Python/Django',
-    pairing_location: 'phone, screenhero'
+    pairing_location: 'phone, screenhero',
```

Comment:
afaik I don't think "phone" is one of the options. It can be. But I think it's Turing, off campus, or remote.